rusk-wallet/src/bin/io/prompt.rs

```diff
@@ -56,80 +75,43 @@ pub(crate) fn create_password(
 ) -> anyhow::Result<Vec<u8>> {
     let pwd = match password.as_ref() {
         Some(p) => p.to_string(),
-        None => {
-            let mut pwd = String::from("");
-
-            let mut pwds_match = false;
-            while !pwds_match {
-                // enter password
-                let q = Question::password("password")
-                    .message("Enter the password for the wallet:")
-                    .mask('*')
-                    .build();
-                let a = requestty::prompt_one(q)?;
-                let pwd1 = a.as_string().expect("answer to be a string");
-
-                // confirm password
-                let q = Question::password("password")
-                    .message("Please confirm the password:")
-                    .mask('*')
-                    .build();
-                let a = requestty::prompt_one(q)?;
-                let pwd2 = a.as_string().expect("answer to be a string");
-
-                // check match
-                pwds_match = pwd1 == pwd2;
-                if pwds_match {
-                    pwd = pwd1.to_string()
-                } else {
-                    println!("Passwords don't match, please try again.");
-                }
-            }
-            pwd
-        }
+        None => create_new_password()?,
     };
 
     Ok(hash(file_version, &pwd))
 }
 
 /// Display the recovery phrase to the user and ask for confirmation
+/// Recursively calls itself until the user confirms they have backed up
 pub(crate) fn confirm_recovery_phrase<S>(phrase: &S) -> anyhow::Result<()>
 where
     S: std::fmt::Display,
 {
     // inform the user about the mnemonic phrase
-    println!("The following phrase is essential for you to regain access to your wallet\nin case you lose access to this computer.");
-    println!("Please print it or write it down and store it somewhere safe:");
-    println!();
-    println!("> {}", phrase);
-    println!();
+    let msg = format!("The following phrase is essential for you to regain access to your wallet\nin case you lose access to this computer. Please print it or write it down and store it somewhere safe.\n> {} \nHave you backed up this phrase?", phrase);
 
     // let the user confirm they have backed up their phrase
-    loop {
-        let q = requestty::Question::confirm("proceed")
-            .message("Have you backed up your recovery phrase?")
-            .build();
-
-        let a = requestty::prompt_one(q)?;
-        if a.as_bool().expect("answer to be a bool") {
-            return Ok(());
-        }
+    let confirm = Confirm::new(&msg)
+        .with_help_message(
+            "It is important you backup the passphrase before proceeding",
```

Comment:
We use `passphrase`, `seed phrase`, `recovery phrase` all over the place. Some crypto wallets also call it `mnemonic`. 

We should standardize around a single term. In the web wallet we use the term `mnemonic`, which is also the most common way it's called in most crypto wallets.

Reply:
Seems like you already fixed this